### ISSUE TITLE
Implement gpu backend for SDL_Render

### ISSUE THREAD

**Issue by flibitijibibo** [thatcosmonaut/SDL#12]
This would be nice to have for Beta but this may end up being how someone reviews the API, so we can wait on this one.

**Comment by Akaricchi** [comment]
Started working on this at https://github.com/taisei-project/SDL/tree/gpu_render

Still very scuffed and broken, but several tests and demos already work. Vulkan-only for now. Going to post a draft PR later to track progress.

**Comment by thatcosmonaut** [comment]
As of #192 this is mostly complete, just needs some performance improvements and testing on a wider variety of clients.

**Comment by flibitijibibo** [comment]
I think the last remaining feature is YUV which Sam offered to look at - there are ways to optimize but at least upstream we've got this in.
